INDEX.md

```diff
@@ -82,6 +92,34 @@ More a GitHub search result than a single repo, Matt Carroll's Flutter UI
 challenges showcase some amazing designs powered by Flutter. Make sure to check
 the accompanying YouTube video so you can watch Matt implement them.
 
+#### [Platform Design](platform_design) _(Flutter team)_
+
+An app designed to look great on both iOS and Android, this sample
+shows how to add platform-specific design touches to an app
+while reusing as much code as possible.
+
+#### [Animation examples](animations) _(Flutter team)_
+
+_currently in development_
```

Comment:
Aren't most of these samples in development?

Reply:
I'm using "in development" to mean "incomplete but not for long."